src/main/java/org/apache/commons/io/input/SequenceReader.java

```diff
@@ -114,9 +115,9 @@ public int read() throws IOException {
      */
     @Override
     public int read(final char[] cbuf, int off, int len) throws IOException {
-        Objects.requireNonNull(cbuf, "cbuf");
-        if (len < 0 || off < 0 || off + len > cbuf.length) {
-            throw new IndexOutOfBoundsException("Array Size=" + cbuf.length + ", offset=" + off + ", length=" + len);
+        IOUtils.checkFromIndexSize(cbuf, off, len);
```

Comment:
Either we should inherit the whole Javadoc or add a real Javadoc to document `@throws` `IndexOutOfBoundsException`.